backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3ApplicationConfiguration.java

```diff
@@ -123,11 +123,11 @@ public void setMaxNetThreads (int maxNetThreads) {
 
 	/** Sets the audio device configuration.
 	 * 
-	 * @param simultaniousSources the maximum number of sources that can be played simultaniously (default 16)
+	 * @param simultaneousSources the maximum number of sources that can be played simultaniously (default 16)
```

Comment:
Still, one usage of "simultaniously" snuck through, heh. Not a big enough problem to worry about.